crypto/cryptlib.c

```diff
@@ -281,22 +311,26 @@ void OPENSSL_showfatal(const char *fmta, ...)
 
             if (!ReportEvent(hEventLog, EVENTLOG_ERROR_TYPE, 0, 0, NULL,
                              1, 0, &pmsg, NULL)) {
-#if defined(DEBUG)
+#   if defined(DEBUG)
```

Comment:
And while you are at it could you replace this with `#   if !defined(NDEBUG)`? We settled for NDEBUG, but this one apparently remained unharmonized.